arrow-gradle-config-kotlin/src/main/kotlin/io.arrow-kt.arrow-gradle-config-kotlin.gradle.kts

```diff
@@ -41,62 +41,77 @@ if (isKotlinMultiplatform) {
       nodejs()
     }
 
+    // TIER 1
     linuxX64()
-
-    mingwX64()
-
-    iosArm32()
-    iosArm64()
+    macosX64()
+    macosArm64()
     iosSimulatorArm64()
     iosX64()
-    macosArm64()
-    macosX64()
-    tvosArm64()
-    tvosSimulatorArm64()
-    tvosX64()
-    watchosArm32()
-    watchosArm64()
+    iosArm64() // soon in tier 1
+    
+    // TIER 2
+    linuxArm64()
     watchosSimulatorArm64()
     watchosX64()
-    watchosX86()
```

Comment:
Should just do a _hard_ remove targets like this 🤔
I can see on sonatype that we get very little downloads for `watchosX86`, but we're getting some... Same for `mingwX64`. These targets are getting < 500 downloads every month, so arguably its all toy projects.

I think there are 3 options:
 - Follow same deprecation - removal process as Kotlin itself
 - Remove in 2.x.x, regardless of what Kotlin is doing
 - Hard remove in 1.2.0

Reply:
Probably `watchosX86` projects are not using that target; they are just targeting all targets that exist as coroutines library does (and other JetBrains libraries).

And that is the real problem, when we remove those targets all of those libraries will break. I would keep this PR as a draft and merge it in the future following Kotlin (and probably coroutines) practices.

Reply:
What do you think about not removing any target, but adding the new ones? They won’t be removed until 1.9 at least.

Reply:
@JavierSegoviaCordoba I think we should merge (and release a new version of this library) now, so we can start targeting those targets in 1.8.

Reply:
Up to you :)

Reply:
If we're not removing anything then it's definitely not an issue. We need to make sure to update the CI workflow of Arrow when we update.

Reply:
> Follow same deprecation - removal process as Kotlin itself
> Remove in 2.x.x, regardless of what Kotlin is doing

Then we can follow either of these two strategies, whichever comes first.